G/Gloo/bundled/patches/mingw-lowercase-include.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/gloo/types.h b/gloo/types.h
```

Comment:
It'd be nice to upstream all the patches

Reply:
Indeed.